Osnova.Net/Osnova.Net.xml

```diff
@@ -27,6 +27,601 @@
             Android = 2
             </summary>
         </member>
+        <member name="F:Osnova.Net.Constants.DtfSubsiteIds.Games">
```

Comment:
Don't forget to fetch from upstream and rebuild the docs before submitting future changes